tests/02_eradiate/01_unit/test_tools/test_regression.py

```diff
@@ -0,0 +1,172 @@
+import numpy as np
+import pytest
+import xarray as xr
+
+import eradiate
+
+eradiate.set_mode("mono")
+
+import mitsuba as mi
+
+import eradiate.test_tools.regression as tt
```

Comment:
isort pass (you might also need to move the `eradiate.set_mode()` call down a bit)